src/aggregate.h

```diff
@@ -279,11 +278,11 @@ class ClassDeclaration : public AggregateDeclaration
     TypeInfoClassDeclaration *vclassinfo;       // the ClassInfo object for this ClassDeclaration
     bool com;                           // true if this is a COM class (meaning it derives from IUnknown)
     bool cpp;                           // true if this is a C++ interface
+    bool objc;                          // true if this is an Objective-C class/interface
```

Comment:
IIRC there is at least one other way to mark a class as c++, it could do with some cleaning up.